src/index.ts

```diff
@@ -225,6 +225,7 @@ const server = createServer(async (request, response) => {
   } catch (err) {
     console.error(err);
     response.statusCode = 500
+    response.write("data: Something went wrong\n\n")
```

Comment:
I'd be curious to know if the error banner shows up in chat because this `data:` packet isn't a JSON string like something expected, and/or because it doesn't end with `data: [DONE]`. Thinking of logic like this:

https://github.com/github/github/blob/f09a9b9dcfc1b7a6a5cf8bfa19587180b54fc363/ui/packages/completions-api/completions-api.ts#L191-L197

https://github.com/github/github/blob/f09a9b9dcfc1b7a6a5cf8bfa19587180b54fc363/ui/packages/completions-api/completions-api.ts#L180

Not that you should necessarily `write` something different here, just thinking about whether your "Something went wrong" is being used or if you could've put _any_ non-JSON value after `data: ` and gotten the same result. 🤷🏻‍♀️

Reply:
I don't think the actual string I'm using here is being used, but it seems like just the presence of error data seems to trigger the error state! I need to find the corresponding code in copilot chat to verify for sure though 👀 

This repo doesn't have tests yet, so I'm thinking of creating an issue and following up on a separate PR to create test scaffolding